src/SimpleStorage.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.17;
+
+import {PluginUUPSUpgradeable, IDAO} from "@aragon/osx/core/plugin/PluginUUPSUpgradeable.sol";
+
+/**
+ * @title Budget
+ * @author DAOBox (Security@DAOBox.app)
+ * @notice Budgeting module for efficient spending from an Aragon OSx DAO using allowance chains
+ * to delegate spending authority
+ */
+contract SimpleStorage is PluginUUPSUpgradeable {
```

Comment:
Would be good to use the same plugin/plugin naming everywhere https://github.com/aragon/osx-plugin-template-hardhat/blob/main/packages/contracts/src/MyPlugin.sol